### PR TITLE
sed defined to enable running bulk load under modern perl

### PR DESCRIPTION
**Pull request by hexylena**
- pg9 → pg16  (and much newer debian)
- fixes yeast loading (again), due to perl deprecating features
- added github action to test PRs

### CODE REVIEW

**Comment by hexylena** [comment]
(it is expected that tests have not run on the branch tip, since the test cases will now only run if .sh/.sql/Dockerfile changes.

File: README.md [hunk 1]
```diff
@@ -56,18 +54,21 @@ docker run -d --name chado-yeast -e INSTALL_YEAST_DATA=1 quay.io/galaxy-genome-a
...
 ```
 
-Given that I as the developer have no easy way to communicate to you as the end user that breaking changes have been made (and keeping backwards compatability is prohibitve for a 1-person team..., sorry!), it is best to pin and read the changelog before upgrading.
+Given that we, as the developers, have no easy way to communicate to you as the end user that breaking changes have been made (and keeping backwards compatability is prohibitve for a small team..., sorry!), it is best to pin and read the changelog before upgrading.
 
 # Changelog
 
+- 2024-02-22
+	- @hexylena updated postgres from 9 to 16
+    - @hexylena rebuilt with latest CSB data which is still quite old.
```
Comment (abretaud) [inline README.md]: tab problem here, mixed spaces/tabs maybe?

**Review by abretaud — APPROVED** [review]
Great to see this update :tada: , thanks @hexylena !

**Comment by hexylena** [comment]
Thanks for the review @abretaud appreciate it!

**Pull request merged**